src/components/Header/Header.css

```diff
@@ -0,0 +1,69 @@
+header {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr;
+  justify-items: center;
+  align-items: center;
+  height: 10vh;
+}
+
+.charcoal {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+  background-color: rgb(38, 70, 83);
+  color: white;
+  text-decoration: none;
+  width: 100%;
+  height: 100%;
+}
+.green {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+  background-color: rgb(42, 157, 143);
+  color: white;
+  text-decoration: none;
+  width: 100%;
+  height: 100%;
+}
+.crayola {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+  background-color: rgb(233, 196, 106);
+  color: white;
+  text-decoration: none;
+  width: 100%;
+  height: 100%;
+}
+.brown {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+  background-color: rgb(244, 162, 97);
+  color: white;
+  text-decoration: none;
+  width: 100%;
+  height: 100%;
+}
+.sienna {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+  background-color: rgb(231, 111, 81);
```

Comment:
The params w/ inline styling are setting the background color. Inline styling overrides this CSS, and the assignment rubric wants it to be that way. Just a friendly heads up, that you don't need this here.  :)